tests/testthat/test-estimate_infections.R

```diff
@@ -117,119 +92,11 @@ test_that("summary with type='parameters' returns all dates by default", {
   expect_equal(unique(summ_filtered$date), target)
 })
 
-test_that("estimate_infections successfully returns estimates using a single breakpoint", {
```

Comment:
Again this is just removing a large number of integraiton tests without justification?